### ISSUE TITLE
copy/pasting auto comments does not work

### ISSUE THREAD

**Issue by yrp604** [Vector35/binaryninja-api#2794]
3136-dev

**Describe the bug**
I have the following basic block:
```
// flag:v,c,z,n set by 2008 @ 0x4a904
2014 @ 0004a90c  w5 = and.d{*}(w2, w5)
2015 @ 0004a910  if (!=) then 2029 @ 0x4a950 else 2042 @ 0x4a914  { uses flag:z from 2014 @ 0x4a90c }
// flag:v,c,z,n set by 2014 @ 0x4a90c
```

I cannot copy/paste the 'flag' comments, and they have a weird empty tooltip

**To Reproduce**
1. Go to'0004a90c
2. Turn on 'Show IL Flag Usage'
3. Attempt to copy a flag string, e.g. the line after 2015
5. Hover over a flag string
6. See error

**Expected behavior**
I can consistently copy/paste text out of binja in a non-surprising manner

**Screenshots**
![image](https://user-images.githubusercontent.com/8716677/146893657-69f24663-1bde-405e-b27f-e17da83f714c.png)

**Version and Platform (required):**
 - Binary Ninja: Dev 2.4.3136-dev
 - OS: Windows
 - Version 11

This binary, rebased to zero is the specific testcase I was using
[libc-2.32.so.txt](https://github.com/Vector35/binaryninja-api/files/7750602/libc-2.32.so.txt)

**Comment by psifertex** [comment]
In graph view, it's totally broken

In linear view, whole-line selection works but partial selection (click-drag) does not.

**Comment by CouleeApps** [comment]
This bug was partially caused by Lifted IL flag annotation lines having bad addresses (their address was an IL index and not a virtual address). There are two different underlying bugs left with copy/paste from this issue:
- [ ] Disassembly lines of width 0 cannot be copied
- [ ] Disassembly view calculates line width by disassembling the instruction at that address

I think having 2 subparts is small enough that this won't turn into a megaissue (tm) and never be addressed, and while your specific bug case is likely solved (in the next success dev build) I'm going to leave this issue open until those two other bugs are addressed.

**Comment by plafosse** [comment]
This bug has been fixed since at lease 4.0
